File: Data structure and algorithms/B2095-白细胞计数.py
n=int(input())
sample=[]
for i in range(n):
    sample.append(float(input()))

sample.remove(max(sample))
sample.remove(min(sample))
average=0.0
for i in range(n-2):
    average+=sample[i]/(n-2)

for i in range(n-2):
    sample[i]=abs(sample[i]-average)
print('%.2f' % average+' '+'%.2f' %max(sample))

File: Exercises/LuoGu/B2095-白细胞计数.cpp
#include<iostream>
#include<vector>
#include<cmath>
using namespace std;

int main(){
    int n;
    cin>>n;
    vector<float> sample(n);
    for(int i=0;i<n;i++){
        cin>>sample[i];
    }
      

    float average=0;
    for(int i=0;i<n;i++){
        average+=sample[i]/5;
    }

}